src/VisualStudio/Xaml/Impl/Implementation/LanguageServer/Handler/Formatting/AbstractFormatDocumentHandlerBase.cs

```diff
@@ -19,7 +19,9 @@ internal abstract class AbstractFormatDocumentHandlerBase<RequestType, ResponseT
         public override bool MutatesSolutionState => false;
         public override bool RequiresLSPSolution => true;
 
+#pragma warning disable IDE0060 // Remove unused parameter 'documentIdentifier' - Unsure whether it can break any code relying on IVTs.
```

Comment:
@jasonmalinowski Do you know if this suppression should remain or otherwise I need a better certain comment.

Reply:
@davidwengier? I wouldn't imagine we have IVTs in our LSP handlers (would seem to defeat the point architecturally), but I never cease to be surprised about our codebase sometimes.

Reply:
This method isn't used by IVT. The pattern for our LSP handlers is to get a language service, in this case the IXamlFormattingService a few lines below, and those cannot be changed (at least not without some careful coordination).

Reply:
Fine by me. Nothing on the VSMac side is using LSP, and I agree with Jason this would be an odd way to do it :)